code/campaignclassic/src/main/res/layout/push_template_filmstrip_carousel.xml

```diff
@@ -66,6 +83,8 @@
         android:id="@+id/manual_carousel_filmstrip_right"
         android:scaleType="centerCrop"
         android:adjustViewBounds="true"
+        android:layout_gravity="center"
```

Comment:
Same comment here, should this be `layout_centerVertical=true`?

Reply:
`android:layout_gravity="center"` is used in combination with `android:layout_centerHorizontal="true"` to center the left and right filmstrip images with the center image